src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -100,7 +100,16 @@ mod private { // this item is private and will not be documented
 }
 ```
 
-`--document-private-items` documents all items, even if they're not public.
+`--document-private-items` causes all items to be documented as if they were `pub`.
+
+## `--document-hidden-items`: Show items that are `#[doc(hidden)]`
+
+By default, `rustdoc` does not document items that are annotated with
+[`#[doc(hidden)]`](write-documentation/the-doc-attribute.html#hidden).
+
+`--document-hidden-items` causes all items to be documented as if they did not have `#[doc(hidden)]`.
```

Comment:
I think you can remove this paragraph, it's what is said in the first line of this diff already so no need for repetition.

Reply:
You mean the heading?  I'm following the existing style of other sections, which is that the body of a section should make sense without its heading.